display_functions.py

```diff
@@ -0,0 +1,15 @@
+
+from get_functions import * 
+
+
+def color_pair_to_string(major_color, minor_color):
+  return f'{major_color}  |  {minor_color}'
+
+def color_code_manual():
```

Comment:
most functions have a test, but not `color_code_manual`. Think of a test for this function.